src/service/video-manager-impl.js

```diff
@@ -327,6 +330,25 @@ class VideoEntry {
   }
 
   /**
+   * Creates a mask to overlay on top of an autoplay video to detect the first
+   * user tap.
+   * We have to do this since many players are iframe-based and we can not get
+   * the click event from the iframe.
+   * We also can not rely on hacks such as constantly checking doc.activeElement
```

Comment:
I'm (positively) surprised that this works :)

So, the user click on the parent frame, can unmute a video via postMessage?